Dockerfile

```diff
@@ -31,22 +30,28 @@ FROM elixir-builder as app-builder
 
 ENV LANG="C.UTF-8" MIX_ENV=prod
 
+RUN apk add --no-cache --update curl
+
 WORKDIR /root
 
+RUN curl https://truststore.pki.rds.amazonaws.com/global/global-bundle.pem \
+    -o aws-cert-bundle.pem
```

Comment:
question: I don't suppose there's a SHA256 sum or something we can use to avoid a supply chain attack like we had with Codecov?

Reply:
Good question... I'm not seeing anything on AWS's site. Of course I could just download it on my own machine, look at the checksum, and hardcode it into the Dockerfile, but that's still dependent on `truststore.pki.rds.amazonaws.com` having a legitimate certificate when I do the download, so I'm not sure how much extra security that buys us.

Reply:
True, though we do do deploys (and therefore rebuild the image) fairly often. Thinking about it from that angle I guess it would be a good thing to have such a check.

Reply:
Added a check, and verified if I replaced the correct SHA256 sum with some gibberish and tried to build locally it caused the build to fail, rather than proceeding along silently.